N23: Add opencombat game (#678)
rust-gamedev/rust-gamedev.github.io: content/news/023/index.md

## Game Updates

### [Open Combat][opencombat]

> ![alt text](open_combat.jpg)

Reviewer: ```suggestion
![Open Combat logo](open_combat.jpg)
```

[Open Combat](https://opencombat.bux.fr/)
is a real time tactical game directly inspired from
[Close Combat Series](https://en.wikipedia.org/wiki/Close_Combat_(series)).

> The player take control of soldier units and order them to win the battles

Reviewer: ```suggestion
The player takes control of soldier units and orders them to win the battles
```

from a top down 2D view.

> Project recently started with Rust language after a Python language proof of
> concept in 2017. You can find a

Reviewer: ```suggestion
The project recently started with Rust language after a Python language proof of
concept in 2017. More info is available at [opencombat.bux.fr][opencombat],
and you can find a
```

[presentation](https://www.youtube.com/watch?v=pxVgjBKXlIw) video and a game

> making of [video](https://www.youtube.com/watch?v=3TOEZ7krhvI).

Reviewer: ```suggestion
making of [video](https://www.youtube.com/watch?v=3TOEZ7krhvI) on YouTube.
```

_Discussions: [r/OpenCombatGame](https://www.reddit.com/r/OpenCombatGame/),
[Discord](https://discord.gg/YD2V7XsBQZ),

> [Forum](https://discourse.opencombat.bux.fr/)_,
> Website: [opencombat.bux.fr](https://opencombat.bux.fr)

Reviewer: ```suggestion
[Forum](https://discourse.opencombat.bux.fr/)_
```
